tests/foreman/cli/test_repository.py

```diff
@@ -691,3 +700,177 @@ def test_positive_upload_content(self):
             "Successfully uploaded file '{0}'".format(RPM_TO_UPLOAD),
             result[0]['message'],
         )
+
+
+class RepositoryExportTestCase(CLITestCase):
+    """Tests for exporting a repository via CLI"""
+
+    export_dir = None
+    is_set_up = False
+    org = None
+
+    def setUp(self):
+        """Create a directory for export, configure permissions and satellite
+        settings
+        """
+        super(RepositoryExportTestCase, self).setUp()
+
+        if not RepositoryExportTestCase.is_set_up:
+            RepositoryExportTestCase.export_dir = gen_string('alphanumeric')
+
+            # Create a new 'export' directory on the Satellite system
+            result = ssh.command('mkdir /mnt/{0}'.format(self.export_dir))
+            self.assertEqual(result.return_code, 0)
+
+            result = ssh.command(
+                'chown foreman.foreman /mnt/{0}'.format(self.export_dir))
+            self.assertEqual(result.return_code, 0)
+
+            result = ssh.command(
+                'ls -Z /mnt/ | grep {0}'.format(self.export_dir))
+            self.assertEqual(result.return_code, 0)
+            self.assertGreaterEqual(len(result.stdout), 1)
+            self.assertIn('unconfined_u:object_r:mnt_t:s0', result.stdout[0])
+
+            # Fix SELinux policy for new directory
+            result = ssh.command(
+                'semanage fcontext -a -t foreman_var_run_t "/mnt/{0}(/.*)?"'
+                .format(self.export_dir)
+            )
+            self.assertEqual(result.return_code, 0)
+
+            result = ssh.command(
+                'restorecon -Rv /mnt/{0}'.format(self.export_dir))
```

Comment:
This is not required once the SELinux bug is fixed. This is a temporary workaround now.